frontend/src/individual_sellingform.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+import { makeStyles, TextField } from "@material-ui/core";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { Autocomplete } from "@material-ui/lab";
+import DateFnsUtils from "@date-io/date-fns";
+
+import {
+  Genebank,
+  herdLabel,
+  Individual,
+  LimitedHerd,
+} from "./data_context_global";
+
+const useStyles = makeStyles({
+  inputField: {
+    margin: "1em 0",
+  },
+});
+
+export function IndividualSellingform({
+  individual,
+  herdOptions,
+  herdKey,
```

Comment:
I'm not thrilled with these names. And what is `herdKey` here?

Reply:
`herdOptions` is a way of naming that has been present in the code base since I started working with it, there are `colorOptions`, `sexOptions` and maybe some more in similar use cases, so I'd rather not change it here. 

The `herdKey` is used to reset the Autocomplete from Material UI when the selling-form-component is rendered in `individual_add`. To clear the Autocomplete when the user clicks on "Ny kanin", I change the key to cause a rerender of the component. It was the only  way I managed to clear the Autocomplete and make it show the placeholder text ("Välj besättning") and not just "undefined". Found here: https://stackoverflow.com/questions/59790956/material-ui-autocomplete-clear-value Maybe there is a better way, but I tried a lot. 
I'm open to a name it differently but thought this was most appropriate. What would you suggest?

Reply:
Ok, good enough.